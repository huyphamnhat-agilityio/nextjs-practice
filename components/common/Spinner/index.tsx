const Spinner = () => {
  return (
    <div
      className="w-12 h-12 rounded-full animate-spin
              border-y-2 border-solid border-primary border-t-transparent"
    ></div>
  );
};

export default Spinner;
