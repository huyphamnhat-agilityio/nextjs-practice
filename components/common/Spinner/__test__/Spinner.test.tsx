import { render } from "@testing-library/react";
import Spinner from "..";

describe("Spinner test cases", () => {
  const setup = () => render(<Spinner />);
  it("should render correctly", () => {
    const { asFragment } = setup();

    expect(asFragment()).toMatchSnapshot();
  });
});
