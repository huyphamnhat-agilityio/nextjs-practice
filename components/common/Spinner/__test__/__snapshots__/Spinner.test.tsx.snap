// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner test cases should render correctly 1`] = `
<DocumentFragment>
  <div
    class="w-12 h-12 rounded-full animate-spin border-y-2 border-solid border-primary border-t-transparent"
  />
</DocumentFragment>
`;
